actions.go

```diff
@@ -0,0 +1,247 @@
+	Name        string          `json:"name"`
+	Description *graphql.String `json:"description,omitempty"`
+	Owner       graphql.ID      `json:"ownerId"`
+	Action      *graphql.ID     `json:"actionId,omitempty"`
+	Filter      *graphql.ID     `json:"filterId,omitempty"`
+	// This is being explictly left out to reduce the complexity of the implementation
+	// action *CustomActionsWebhookActionCreateInput
+}
+
+type CustomActionsTriggerDefinitionUpdateInput struct {
+	Id          graphql.ID      `json:"id"`
+	Name        *graphql.String `json:"name,omitempty"`
+	Description *graphql.String `json:"description,omitempty"`
+	Owner       *graphql.ID     `json:"ownerId,omitempty"`
+	Action      *graphql.ID     `json:"actionId,omitempty"`
+	Filter      *graphql.ID     `json:"filterId,omitempty"`
+}
+
+func (client *Client) CreateWebhookAction(input CustomActionsWebhookActionCreateInput) (*CustomActionsExternalAction, error) {
+	var m struct {
+		Payload struct {
+			WebhookAction CustomActionsExternalAction
+			Errors        []OpsLevelErrors
+		} `graphql:"customActionsWebhookActionCreate(input: $input)"`
+	}
+	v := PayloadVariables{
+		"input": input,
+	}
+	err := client.Mutate(&m, v)
+	return &m.Payload.WebhookAction, HandleErrors(err, m.Payload.Errors)
+}
+
+// TODO: Not implemented in the API yet
+//func (client *Client) GetCustomAction(input IdentifierInput) (*CustomActionsTriggerDefinition, error) {
+
+//}
+
+func (client *Client) ListCustomActions(variables *PayloadVariables) (CustomActionsExternalActionsConnection, error) {
+	var q struct {
+		Account struct {
+			Actions CustomActionsExternalActionsConnection `graphql:"customActionsExternalActions(after: $after, first: $first)"`
+		}
+	}
+	if variables == nil {
+		variables = &PayloadVariables{
+			"after": "",
+			"first": client.pageSize,
+		}
+	}
+	if err := client.Query(&q, nil); err != nil {
+		return CustomActionsExternalActionsConnection{}, err
+	}
+	for q.Account.Actions.PageInfo.HasNextPage {
+		(*variables)["after"] = q.Account.Actions.PageInfo.End
+		resp, err := client.ListCustomActions(variables)
+		if err != nil {
+			return CustomActionsExternalActionsConnection{}, err
+		}
+		q.Account.Actions.Nodes = append(q.Account.Actions.Nodes, resp.Nodes...)
+		q.Account.Actions.PageInfo = resp.PageInfo
+	}
+	return q.Account.Actions, nil
+}
+
+func (client *Client) UpdateWebhookAction(input CustomActionsWebhookActionUpdateInput) (*CustomActionsExternalAction, error) {
+	var m struct {
+		Payload struct {
+			WebhookAction CustomActionsExternalAction
+			Errors        []OpsLevelErrors
+		} `graphql:"customActionsWebhookActionUpdate(input: $input)"`
+	}
+	v := PayloadVariables{
+		"input": input,
+	}
+	err := client.Mutate(&m, v)
+	return &m.Payload.WebhookAction, HandleErrors(err, m.Payload.Errors)
+}
+
+func (client *Client) DeleteWebhookAction(input IdentifierInput) error {
+	var m struct {
+		Payload ResourceDeletePayload `graphql:"customActionsWebhookActionDelete(resource: $input)"`
```

Comment:
Updated this to use resource for the input payload.  Not sure if we want to ask dev team to change this to input to be more consistent with other mutations.